### PR TITLE
larger page buttons in Table footer

### PR DESCRIPTION
**Pull request by potluck**
**Jira:** https://clever.atlassian.net/browse/CLASSROOMS-1450

**Overview:** The buttons to navigate through pages of the `Table` component are pretty small. Make them a bit bigger.

**Screenshots/GIFs:** 
This gif involves me tabbing through a page with the new table footer (initial), and then one with the old footer with smaller buttons for comparison.
bigger-footer-buttons

### CODE REVIEW

File: src/Table/Footer.less [hunk 1]
```diff
@@ -9,7 +9,7 @@
 }
 
 .Table--footer--cell {
-  font-size: @size_s;
+  font-size: @size_m;
```
Comment (osdiab) [inline src/Table/Footer.less]: Maybe use the type size utilities instead here: https://github.com/Clever/components/blob/master/src/less/type-size.less

**Comment by CarlNelson** [comment]
My suggestion is to instead turn the buttons into link buttons instead of linkPlain.

This makes the click target larger without increasing the type size

**Comment by potluck** [comment]
link-buttons

**Comment by potluck** [comment]
Updated ^

**Comment by potluck** [comment]
I gotta bounce so will figure out CI stuff later, but let me know. Prob still wanna make the link button smaller

**Comment by potluck** [comment]
link-buttons

**Comment by potluck** [comment]
@CarlNelson let me know what you think ^
IMO I think it would still be nicer if the text was a bit larger. The footer at the bottom of the apps list in the schools dashboard just feels *so small* compared to the size of the table. On top of that it's at the bottom of the page, which I think further contributes to feeling like it could be bigger.

File: src/Table/Footer.less [hunk 2]
```diff
@@ -9,7 +10,7 @@
 }
 
 .Table--footer--cell {
-  font-size: @size_s;
+  .text--small;
```
Comment (CarlNelson) [inline src/Table/Footer.less]: Do we need this if we're using the buttons?

**Comment by CarlNelson** [comment]
@potluck this looks good to me. The type size for these buttons shouldn't be larger than the text in the table as the table data should be the primary visually. 

Otherwise, ship it.

**Pull request merged**